include/roamer.h

```diff
@@ -1,17 +1,21 @@
 #ifndef GUARD_ROAMER_H
 #define GUARD_ROAMER_H
 
-void ClearRoamerData(void);
-void ClearRoamerLocationData(void);
+void DeactivateAllRoamers(void);
 void InitRoamer(void);
 void UpdateLocationHistoryForRoamer(void);
-void RoamerMoveToOtherLocationSet(void);
-void RoamerMove(void);
-bool8 IsRoamerAt(u8 mapGroup, u8 mapNum);
-void CreateRoamerMonInstance(void);
+void RoamerMoveToOtherLocationSet(u8 roamerIndex);
```

Comment:
```suggestion
void RoamerMoveToOtherLocationSet(u32 roamerIndex);
```
I wrote this before I knew using 32 bit types is (slightly) faster, you might wanna change all of these.